.github/workflows/tests.yml

```diff
@@ -29,24 +29,18 @@ jobs:
         - ["3.11", "py311-integration", "ubuntu-latest"]
         - ["3.12", "py312-integration", "ubuntu-latest"]
         - ["3.13", "py313-integration", "ubuntu-latest"]
+        - ["3.9", "buildout", "ubuntu-latest"]
+        - ["3.13", "buildout", "ubuntu-latest"]
         - ["3.10", "coverage", "ubuntu-latest"]
 
     runs-on: ${{ matrix.config[2] }}
     name: ${{ matrix.config[1] }}
     steps:
-    - uses: actions/checkout@v4.2.2
+    - uses: actions/checkout@v5.0.0
```

Comment:
By using `@v5` you will be getting minor and patch fixes, may I gather that you want specific versions due to the complexity of this distribution? 🤔

Reply:
Using a tag should be more secure.  We might want to do that in more places.  I thought Zope was doing that in some repos, but I can't find it.

Anyway, we were using tags until now in this package, so I say we keep them.  But I don't feel strongly about it.